xhost.mk

```diff
@@ -0,0 +1,48 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS    += xhost
+XHOST_VERSION := 1.0.8
+DEB_XHOST_V   ?= $(XHOST_VERSION)
+
+xhost-setup: setup
+	wget -q -nc -P $(BUILD_SOURCE) https://www.x.org/archive//individual/app/xhost-$(XHOST_VERSION).tar.gz{,.sig}
+	$(call PGP_VERIFY,xhost-$(XHOST_VERSION).tar.gz)
+	$(call EXTRACT_TAR,xhost-$(XHOST_VERSION).tar.gz,xhost-$(XHOST_VERSION),xhost)
+
+ifneq ($(wildcard $(BUILD_WORK)/xhost/.build_complete),)
+xhost:
+	@echo "Using previously built xhost."
+else
+xhost: xhost-setup libx11 libxau libxmu xorgproto xxhash
+	cd $(BUILD_WORK)/xhost && ./configure -C \
+		--host=$(GNU_HOST_TRIPLE) \
+		--prefix=/usr \
```

Comment:
$(MEMO_PREFIX)$(MEMO_SUB_PREFIX) for --prefix

Reply:
Need --build too

Reply:
`--build=$$($(BUILD_MISC)/config.guess) \`
`--prefix=$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)`

--build should be the the top flag